istioctl/pkg/describe/describe.go

```diff
@@ -107,7 +107,7 @@ func podDescribeCmd(ctx cli.Context) *cobra.Command {
 		Short:   "Describe pods and their Istio configuration [kube-only]",
 		Long: `Analyzes pod, its Services, DestinationRules, and VirtualServices and reports
 the configuration objects that affect that pod.`,
-		Example: `  istioctl experimental describe pod productpage-v1-c7765c886-7zzd4`,
+		Example: `  istioctl experimental describe pod <pod-name>[.<namespace>]`,
```

Comment:
Thank you! Can you also update line 105 `Use: "pod <pod>",` to be more helpful?

Reply:
> Thank you! Can you also update line 105 `Use: "pod <pod>",` to be more helpful?

Solved